movo_common/movo_third_party/eband_local_planner/CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Changelog for package eband_local_planner
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.4.0 (2020-02-14)
+------------------
+* Adds Dynamic Reconfigure for planner parameters 
```

Comment:
Is this an issue that it says ROS melodic and not noetic?

Reply:
this is a third party pkg. its hasnt been updated in a while but hopefully it will work for noetic. heres the repo https://github.com/utexas-bwi/eband_local_planner

last push was made pre-noetic (3 yrs ago)